<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalogToolOptionDialog</class>
 <widget class="QDialog" name="AnalogToolOptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AnalogToolOptionDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0" columnstretch="2,1">
   <item row="0" column="0" rowspan="3">
    <widget class="QGroupBox" name="analogsGroupBox">
     <property name="title">
      <string>Analog channels</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="processingGroupBox">
     <property name="title">
      <string>Data processing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="overwriteAnalogsButton">
        <property name="text">
         <string>Overwrite analog channels data</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="createAnalogsButton">
        <property name="text">
         <string>Create new analog channels</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="styleSheet">
      <string notr="true">QToolBox::tab {
  background-image: url(:/Resources/Images/disclosureTriangleSmallRightBlack.png);
  background-repeat: no-repeat;
  background-position: center left;
  padding-left:12px;
}

QToolBox::tab:selected {
  background-image: url(:/Resources/Images/disclosureTriangleSmallDownBlack.png);
}

QScrollArea {background-color:transparent;}

QScrollBar:horizontal {
  height: 0px;
  width: 0px;
}
     
QScrollBar:vertical {
  height: 0px;
  width: 0px;
}
</string>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AnalogToolOptionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AnalogToolOptionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
